### PR TITLE
NO-ISSUE Refactor kubeapi sub test InfraEnv condition

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

**Review by flaper87 — APPROVED** [review]
thanks! :spaghetti:

File: subsystem/kubeapi_test.go [hunk 1]
```diff
@@ -240,6 +240,16 @@ func checkClusterCondition(ctx context.Context, key types.NamespacedName, condit
 	}, "2m", "2s").Should(Equal(reason))
 }
 
+func checkInfraEnvCondition(ctx context.Context, key types.NamespacedName, message string) {
+	Eventually(func() string {
+		condition := conditionsv1.FindStatusCondition(getInfraEnvCRD(ctx, kubeClient, key).Status.Conditions, v1beta1.ImageCreatedCondition)
```
Comment (YuviGold) [inline subsystem/kubeapi_test.go:245]: It's hard to infer from the function name that `ImageCreatedCondition` condition is being checked. So either to change the name of the function or to receive the condition as a parameter as well (which would be more generic)

Comment (rollandf) [reply subsystem/kubeapi_test.go:245]: Currently there is only one condition on InfraEnv.
If we will add more, we could add a parameter. WDYT?

Comment (YuviGold) [reply subsystem/kubeapi_test.go:245]: I understand that, but I still think it shouldn't be hidden.

Comment (rollandf) [reply subsystem/kubeapi_test.go:245]: Ack, I will refactor in #1550

**Pull request merged**